cmd/loadTests.go

```diff
@@ -97,17 +97,17 @@ func setup(cmd *cobra.Command, args []string) {
 	if len(token) == 0 {
 		token, err = auth.GetTokenFromOC()
 		if err != nil {
-			tokenRequestURI, err := auth.GetTokenRequestURI(framework.CommonController.KubeRest()) // authorization.FindTokenRequestURI(framework.CommonController.KubeRest())
+			tokenRequestURI, err := auth.GetTokenRequestURI(framework.Cluster.CommonController.KubeRest()) // authorization.FindTokenRequestURI(framework.CommonController.KubeRest())
 			if err != nil {
 				klog.Fatalf("a token is required to capture metrics, use oc login to log into the cluster: %v", err)
 			}
 			klog.Fatalf("a token is required to capture metrics, use oc login to log into the cluster. alternatively request a token and use the token flag: %v", tokenRequestURI)
 		}
 	}
 
-	metricsInstance := metrics.NewEmpty(term, framework.CommonController.KubeRest(), 5*time.Minute)
+	metricsInstance := metrics.NewEmpty(term, framework.Cluster.CommonController.KubeRest(), 5*time.Minute)
```

Comment:
@sawood14012 I wasn't sure about which kind of client (i.e. appstudio/appstudio-user/cluster) would be used for different function calls in this file after migration to kcp, so I just guessed which ones should be used.

Feel free to add a suggestion if some of them are wrong. Or if there should be a separate jira ticket created for handling this. Thanks.